src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -100,10 +100,7 @@ public override async Task<bool> Execute()
 
                 if (!this.InstallerUrls.Any())
                 {
-                    this.InstallerUrls = PromptProperty(
-                        new Installer(),
-                        this.InstallerUrls,
-                        nameof(Installer.InstallerUrl));
+                    PromptHelper.PromptProperty(this, nameof(this.InstallerUrls), this.InstallerUrls, minimum: 1, validationModel: new Installer(), validationName: nameof(Installer.InstallerUrl));

```

Comment:
>PromptProperty [](http://example.com/codeflow?start=33&length=14)

Doesn't this still set the value too?

Reply:
Yes you're right, I wanted the name to be as concise as possible, but I think setting the value is an important behavior to note. Reverted to "PromptPropertyAndSetValue"